### ISSUE TITLE
The demo does not work because of the new insecure origins policy

### ISSUE THREAD

**Issue by cotejp** [43081j/funky#4]
The getUserMedia() function no longer works when hosted via insecure origins (non-https).

**Comment by thomasmaclean** [comment]
Just a quick note that you can still see the demo if you clone the repo and view it locally. As long as you have Node.js installed you can do the following in the funky directory:

`npm install http-server -g`
`http-server`
go to 127.0.0.1:8080

**Comment by aotaduy** [comment]
I will try that, but will be cool to have the demo online

Thanks!

**Comment by purestudio** [comment]
hi, 
is there any way to make this demo working ? 
sorry i didn't understand thomasmaclean solution.

Thanks

**Comment by piotr-galas** [comment]
@purestudio 

### running locally
To make it work locally you need to have a server running. It could be a node server like @thomasmaclean suggested or any other server.
If you have PHP installed it could be a command `php -S localhost:8080` as well

Then you need to navigate to `http://localhost:8080/examples`

## make it work locally

This library is using old deprecated API to make it work you need to change those lines
![img](https://p58.f1.n0.cdn.getcloudapp.com/items/ApujDlK6/Screenshot%202020-05-08%20at%2010.28.36.png?v=ab221061095ee2309282203645c0483f)

I can prepare PR, but author needs to approve it.  Because it looks like project is abandoned I will not do it. 
Will you approve my PR @43081j

**Comment by 43081j** [comment]
yup, sorry for not keeping up with this, its one of a couple of aged repos now i haven't had any time to properly maintain.

if you do a PR i will be happy to review it

**Comment by samyk** [comment]
Fixed in 68f86a3 which has been merged into matser
